### PR TITLE
Add ccminer build to v0.2.0

### PR DESCRIPTION
**Pull request by jcvernaleo**

### CODE REVIEW

**Comment by alexlyp** [comment]
OK

File: VERSION [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "miners": {
     "version": "0.2.0",
     "build": "",
-    "date": "2016-06-27",
+    "date": "2016-06-28",
```
Comment (jolan) [inline VERSION]: Just noticed that the date should be July not June

Comment (jcvernaleo) [inline VERSION]: You should be happy I got the year right, I'm still getting that wrong on checks :)

**Comment by jolan** [comment]
OK after fixing the month.

**Pull request merged**